tests/CloseConflictingAppsTests.js

```diff
@@ -112,38 +107,43 @@ gpii.tests.conflictingApps.testDefs = [
                 "com.freedomscientific.jaws": [
```

Comment:
I know it's something we do a lot, but I really dislike this pattern of using real-looking SR data in tests that aren't really testing anything specific to the SR entry.   I would strongly prefer having purely synthetic SR data here, like `gpii.test.conflicting.solutionOne`, etc.

Reply:
This is probably true, however, for the intention of the PR and it's usage I may just leave them as is unchanged for now.

Reply:
It's fine, I'll search for a bunch of variations and clean up when I figure out the proper way I'd like to organise test material in the LSR era.